doc/api/documentation.md

```diff
@@ -29,25 +29,28 @@ or see [the contributing guide][] for directions on how to submit a patch.
 <!--type=misc-->
 
 Throughout the documentation, you will see indications of a section's
-stability.  The Node.js API is still somewhat changing, and as it
-matures, certain parts are more reliable than others.  Some are so
+stability. The Node.js API is still somewhat changing, and as it
+matures, certain parts are more reliable than others. Some are so
 proven, and so relied upon, that they are unlikely to ever change at
-all.  Others are brand new and experimental, or known to be hazardous
+all. Others are brand new and experimental, or known to be hazardous
 and in the process of being redesigned.
 
 The stability indices are as follows:
 
 ```txt
 Stability: 0 - Deprecated
-This feature is known to be problematic, and changes are
-planned.  Do not rely on it.  Use of the feature may cause warnings.  Backwards
-compatibility should not be expected.
+This feature is known to be problematic, and changes are planned. Do
+not rely on it. Use of the feature may cause warnings to be emitted.
+Backwards compatibility should not be expected.
 ```
 
 ```txt
 Stability: 1 - Experimental
-This feature is subject to change, and is gated by a command line flag.
-It may change or be removed in future versions.
+This feature is still under active development and subject to non-backwards
```

Comment:
Yes, this is consistent with the existing language. For instance, under deprecated, it starts with 'This feature' and has `the feature` in the next sentence.